src/vaults/AsyncRequests.sol

```diff
@@ -629,4 +633,11 @@ contract AsyncRequests is BaseInvestmentManager, IAsyncRequests {
 
         return VaultPricingLib.calculatePrice(shareToken, shares, vaultDetails.asset, vaultDetails.tokenId, assets);
     }
+
+    function _mapPoolId(uint64 poolId) internal pure returns (uint64 mappedPoolId) {
+        // TODO: update v2CentrifugeId and add all pools before deployment
+        uint16 v2CentrifugeId = 1;
```

Comment:
NIT. Maybe we want to choose another number, like `uint16(-1)`, to avoid colliding this with Ethereum chain Id. That could help later for mapping this to `wormholeId`

Reply:
The centrifuge ID will just be the one wherever we deploy the legacy pools.